pkg/api/controllers/transaction_controller_test.go

```diff
@@ -270,6 +272,8 @@ func TestGetTransactions(t *testing.T) {
 				})
 				require.Equal(t, http.StatusOK, rsp.Result().StatusCode)
 
+				time.Sleep(time.Second)
```

Comment:
Remove that.

Reply:
This is on purpose. I want to post 3 transactions with different timestamps to be able to test the new filter by date feature. As they are inserted so quickly before, their timestamps were the same. Would you do it differently?

Reply:
Ok, not read the code to be honest. Most of the time, a sleep in tests is a no-go for me.
In this case, you could insert transactions using the store with dates you want and gain no sleep.